src/timeMachine/components/SubmitQueryButton.tsx

```diff
@@ -150,7 +150,11 @@ const mstp = (state: AppState) => {
   const queryStatus = getActiveTimeMachine(state).queryResults.status
 
   const activeQueryText = getActiveQuery(state).text
-  const submitButtonDisabled = activeQueryText === ''
+  const tags = getActiveQuery(state).builderConfig.tags
+  const measurementTag = tags?.[0]
+  const submitButtonDisabled =
+    activeQueryText === '' ||
+    (measurementTag && measurementTag.values.length === 0)
```

Comment:
This adds a condition so that the first tag has to have a selected value in order to be a valid runnable query